pkg/ccl/sqlproxyccl/proxy_handler_test.go

```diff
@@ -88,6 +88,16 @@ func newProxyServer(
 	ln, err := net.Listen("tcp", listenAddress)
 	require.NoError(t, err)
 
+	// Create empty test denylist file if not specified.
+	if opts.Denylist == "" {
```

Comment:
I think it is better to fix the denylist code to not generate warning if a denylist is not set (and not used) instead of generating a temporary file. The warning still should still be logged in case that a file doesn't exist but is set as the denylist source.